battstat/apmlib/apmlib.c

```diff
@@ -267,110 +272,116 @@ int apm_get_events(int fd, int timeout, apm_event_t * events, int n)
+    { APM_DISABLED,        "Power management disabled" },
+    { APM_CONNECTED,       "Real mode interface already connected" },
+    { APM_NOT_CONNECTED,   "Interface not connected" },
+    { APM_16_CONNECTED,    "16 bit interface already connected" },
+/* N/A { APM_16_UNSUPPORTED, "16 bit interface not supported" }, */
+    { APM_32_CONNECTED,    "32 bit interface already connected" },
+    { APM_32_UNSUPPORTED,  "32 bit interface not supported" },
+    { APM_BAD_DEVICE,      "Unrecognized device ID" },
+    { APM_BAD_PARAM,       "Parameter out of range" },
+    { APM_NOT_ENGAGED,     "Interface not engaged" },
 #ifdef APM_BAD_FUNCTION
-	{ APM_BAD_FUNCTION,     "Function not supported" },
+    { APM_BAD_FUNCTION,    "Function not supported" },
 #endif
 #ifdef APM_RESUME_DISABLED
-	{ APM_RESUME_DISABLED,	"Resume timer disabled" },
+    { APM_RESUME_DISABLED, "Resume timer disabled" },
 #endif
-	{ APM_BAD_STATE,	"Unable to enter requested state" },
-/* N/A	{ APM_NO_EVENTS,	"No events pending" }, */
-	{ APM_NOT_PRESENT,	"No APM present" }
+    { APM_BAD_STATE,       "Unable to enter requested state" },
+/* N/A { APM_NO_EVENTS, "No events pending" }, */
+    { APM_NOT_PRESENT,     "No APM present" }
 };
-#define ERROR_COUNT	(sizeof(error_table)/sizeof(lookup_t))
+#define ERROR_COUNT (sizeof(error_table)/sizeof(lookup_t))
 
 /* Return character string describing error messages from APM kernel */
-const char *apm_error_name( unsigned int err )
+const char *
+apm_error_name (unsigned int err)
 {
   int i;
   
-  for(i=0; i<ERROR_COUNT; i++)
-    if(err == error_table[i].key) return(error_table[i].msg);
+  for (i=0; i<ERROR_COUNT; i++)
+    if (err == error_table[i].key) return (error_table[i].msg);
 
   return "Unknown error";
 }
 
-int apm_reject( int fd )
+int
+apm_reject (int fd)
 {
 #ifdef APM_IOC_REJECT
-    if ( ioctl( fd, APM_IOC_REJECT, NULL ) ) 
-	return apm_last_error( fd );
+    if (ioctl fd, APM_IOC_REJECT, NULL )) 
+    return apm_last_error (fd);
     else
 #endif
-	return 0;
+    return 0;
 }
 
-#ifdef APM_IOC_IGNORE		/* detect kernel support of IGNORE/NOIGNORE functions */
-int apm_set_ignore(int fd, int mode)
+#ifdef APM_IOC_IGNORE   /* detect kernel support of IGNORE/NOIGNORE functions */
+int
+apm_set_ignore (int fd, int mode)
 /* Ignore Standby. */
 {
     if (mode == IGNORE)
     {
-	printf("Telling kernel to ignore system standby/suspend mode\n");
-	return ioctl(fd, APM_IOC_IGNORE, NULL);
+    printf ("Telling kernel to ignore system standby/suspend mode\n");
+    return ioctl (fd, APM_IOC_IGNORE, NULL);
     }
     else
     {
-	printf("Telling kernel not to ignore system standby/suspend mode\n");
-	return ioctl(fd, APM_IOC_NOIGNORE, NULL);
+    printf ("Telling kernel not to ignore system standby/suspend mode\n");
+    return ioctl (fd, APM_IOC_NOIGNORE, NULL);
     }
-    printf("NOTE: User-generated suspend/standby requests are not ignored\n");
+    printf ("NOTE: User-generated suspend/standby requests are not ignored\n");
```

Comment:
missing an indent level

Reply:
I do not understand what you expect here?